libraries/AP_TemperatureSensor/AP_TemperatureSensor_MAX31865.cpp

```diff
@@ -94,7 +94,7 @@ void AP_TemperatureSensor_MAX31865::init()
     _dev->set_speed(AP_HAL::Device::SPEED_HIGH);
 
     /* Request 10Hz update */
```

Comment:
Comment needs a update, you might as well go to `200` too, less math is more better.

Reply:
fixed